sdk/python/setup.py

```diff
@@ -101,15 +101,15 @@
     "avro==1.10.0",
     "gcsfs",
     "urllib3>=1.25.4",
-    "pytest==6.0.0",
+    "pytest>=6.0.0",
     "pytest-cov",
     "pytest-xdist",
     "pytest-benchmark>=3.4.1",
     "pytest-lazy-fixture==0.6.3",
     "pytest-timeout==1.4.2",
     "pytest-ordering==0.6.*",
     "pytest-mock==1.10.4",
-    "Sphinx!=4.0.0",
+    "Sphinx!=4.0.0,<4.4.0",
```

Comment:
was this Sphinx change intentional?

Reply:
Yes - without this restriction `pip-tools` was unable to find a valid set of dependencies for Python 3.7 🤷